RSDKv5/RSDK/Input/Wii/WiiInputDevice.cpp

```diff
@@ -129,12 +131,15 @@ void RSDK::SKU::InputDeviceWii::ProcessInput(int32 controllerID) {
 }
 
 void RSDK::SKU::InputDeviceWii::CloseDevice() {
+    //printf("CloseDevice(): closing wii input device %i\r");
     this->active     = false;
     this->isAssigned = false;
-    WPAD_Shutdown();
```

Comment:
Could this be readded to `RenderDevice::ProcessEvents()` maybe? (after setting `isRunning = false`, which is essentially triggering an exit)

Reply:
I did what the SDL2 input backend does and added a `ReleaseWiiInputAPI` function that gets called when the engine exits.